README.md

```diff
@@ -1 +1,77 @@
-# hello-microverse
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# hello-microverse
+
+> This is a site showing hello microverse text as a header.
```

Comment:
I know you did a good effort creating your description for your project, but I strongly recommend you adding a much better description, in you want a reference, 3 or more lines can be good, you can explain what you did, what kind of technologies you used, what is the scope, and so on.

- [ ] Improve your description for your project.